test/SpagoSpec.hs

```diff
@@ -420,6 +420,28 @@ spec = around_ setup $ do
       cd ".."
       testdir "output" `shouldReturn` False
 
+    describe "import checking" $ do
+
+      it "Spago should fail on direct project imports from transitive dependencies" $ do
+        spago ["init"] >>= shouldBeSuccess
+        rm "spago.dhall"
+        writeTextFile "spago.dhall" $ "{ name = \"check-imports\", dependencies = [\"effect\"], packages = ./packages.dhall }"
+        rm "src/Main.purs"
+        writeTextFile "src/Main.purs" "module Main where\nimport Prelude\nmain = unit"
+        rm "test/Main.purs"
+        spago ["build"]
```

Comment:
`spago build` was spitting out all of the "Compiling X" messages, and I didn't want to capture all that in the fixtures. I edited the test to run the build once, then run it again and capture the smaller error/warning (but I'm open to changing it to call `spago build` once, and I can sit down and figure out the fixtures)

Reply:
There’s also the —quiet flag; would that help?

Reply:
I think the pattern "call `build` once and then call again to get small output" is generally what we use in many other tests, so that sounds good 👍